src/main/java/io/jenkins/plugins/eggplant/utils/CLIRunnerHelper.java

```diff
@@ -33,6 +38,7 @@
   private FilePath workspace;
   private String cliFilename;
   private FilePath cliFilePath;
+  private Proxy proxy;
```

Comment:
One thing that i'm curious is that will this variable 'proxy' being used elsewhere?
If not, i think we can make setProxy returns a value instead right?